channel/state/outcome/guarantee.go

```diff
@@ -7,8 +7,10 @@ import (
 
 // A Guarantee is an Allocation with AllocationType == GuaranteeAllocationType and Metadata = encode(GuaranteeMetaData)
 type GuaranteeMetadata struct {
-	Left  types.Destination // The peer who plays the role of Alice (peer 0)
-	Right types.Destination // The peer who plays the role of Bob (peer n+1, where n=len(intermediaries))
+	// The peer who plays the role of Alice (peer 0)
+	Left types.Destination
+	// The peer who plays the role of Bob (peer n+1, where n=len(intermediaries)
```

Comment:
This comment confuses me: in the virtual funding protocol, is there a guarantee that has `left=Alice` and `right=Bob`.

Reply:
(This commit is accidentally part of this PR.)

Agreed that that comment is confusing, and in particular it does not reflect existing usage of this struct in the codebase. I had gotten confused on some overloading of leader / follower, left / right, etc, and intended to attach better godoc comments to existing structs.